src/time/YamlDateTime.cc

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "YamlDateTime.h"
+
+#include <ctime>
+#include <iomanip>
+#include <ostream>
+
+void
+Time::YamlDateTime::print(std::ostream &os) const
+{
+    // need to add fractions and timezone on top of this
+    static const char *yaml_time_format = "%Y-%m-%d %H:%M:%S";
```

Comment:
I do not see why we should name this format, but if you insist on naming it, please do not make the variable static and modifiable (there is no reason for any of that).

```suggestion
    const auto yaml_time_format = "%Y-%m-%d %H:%M:%S";
```